content/applications/finance/accounting/customer_invoices.rst

```diff
@@ -4,225 +4,230 @@
+   - :doc:`payments`
+   - :doc:`bank/reconciliation`
 
-.. image:: customer_invoices/invoice05.png
+.. _accounting/invoice/followup:
 
-A typical journal entry generated from a validated invoice will look
-like as follows:
-
-+-----------------------+---------------+----------------+-------------+--------------+
-| **Account**           | **Partner**   | **Due date**   | **Debit**   | **Credit**   |
-+=======================+===============+================+=============+==============+
-| Accounts Receivable   | Agrolait      | 01/07/2015     | 115         |              |
-+-----------------------+---------------+----------------+-------------+--------------+
-| Taxes                 | Agrolait      |                |             | 15           |
-+-----------------------+---------------+----------------+-------------+--------------+
-| Sales                 |               |                |             | 100          |
-+-----------------------+---------------+----------------+-------------+--------------+
-
-Payment
--------
-
-In Odoo, an invoice is considered to be paid when the associated
-accounting entry has been reconciled with the payment entries. If there
-has not been a reconciliation, the invoice will remain in the Open state
-until you have entered the payment.
-
-A typical journal entry generated from a payment will look like as
-follows:
-
-+-----------------------+---------------+----------------+-------------+--------------+
-| **Account**           | **Partner**   | **Due date**   | **Debit**   | **Credit**   |
-+=======================+===============+================+=============+==============+
-| Bank                  | Agrolait      |                | 115         |              |
-+-----------------------+---------------+----------------+-------------+--------------+
-| Accounts Receivable   | Agrolait      |                |             | 115          |
-+-----------------------+---------------+----------------+-------------+--------------+
-
-Receive a partial payment through the bank statement
-----------------------------------------------------
+Payment follow-up
+=================
 
-You can manually enter your bank statements in Odoo, or you can import
-them in from a csv file or from several other predefined formats
-according to your accounting localisation.
+Odoo helps define a :doc:`follow-up <payments/follow_up>` strategy. Different actions can be set up
+to remind customers to pay their outstanding invoices, depending on how much the customer is
+overdue. These actions are bundled into follow-up levels that trigger when an invoice is overdue by
+a certain number of days. If there are other overdue invoices for the same customer, the actions are
+performed on the most overdue invoice.
 
-Create a bank statement from the accounting dashboard with the related
-journal and enter an amount of $100 .
+.. _accounting/invoice/followup-reports:
 
-.. image:: customer_invoices/invoice06.png
+Follow-up Reports
```

Comment:
This section seems like it repeats a lot of information that's already covered on the follow-ups page itself. I'll leave it for now, but let's ask Audrey what she thinks.

Reply:
@dade-odoo You're right, I'll get rid of this section and just keep the explanation above where I referenced the "Follow-up on invoices" page. Thanks!